Build.cmd

```diff
@@ -1,2 +1,2 @@
 @echo off
-powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\build.ps1""" -build -restore %*"
\ No newline at end of file
+powershell -ExecutionPolicy ByPass -NoProfile -command "Measure-Command { & """%~dp0eng\build.ps1""" -build -restore %* } | Select-Object TotalMinutes, TotalSeconds"
```

Comment:
Are we keeping this change?
( I do not mind, I have similar wrappers in local profile anyway...)

Reply:
We most certainly are. it's good :-)